modules/log-collector-rsyslog-server.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log-collector-rsyslog-server_{context}"]
+= Configuring the collector to listen for connections as a syslog server
+
+You can configure your log collector to collect journal format infrastructure logs from outside the cluster by specifying `syslog` as a receiver input in the `ClusterLogForwarder` custom resource (CR). This enables you to use a single common log store for journal format infrastructure logs that are collected from both inside and outside of your {product-title} cluster.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {clo}.
+* You have created a `ClusterLogForwarder` CR.
+
+.Procedure
+
+. Modify the `ClusterLogForwarder` CR to add configuration for the `syslog` receiver input:
++
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1beta1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+  serviceAccountName: <service_account_name>
+  inputs:
+    - name: syslog-receiver # <1>
+      receiver:
+        type: syslog # <2>
+        syslog:
+          port: 10514 # <3>
+  pipelines: # <4>
+    - name: syslog-pipeline
+      inputRefs:
+        - syslog-receiver
+# ...
+----
+<1> Specify a name for your input receiver.
+<2> Specify the input receiver type as `syslog`.
+<3> Optional: Specify the port that the input receiver listens on. This must be a value between `1024` and `65535`. The default value is `10514` if this is not specified.
+<4> Configure a pipeline for your input receiver.
+
+. Apply the changes to the `ClusterLogForwarder` CR:
```

Comment:
Small nit to explicitly mention running the command [per doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command):
```suggestion
. Apply the changes to the `ClusterLogForwarder` CR by running the following command:
```

Reply:
Thank you! Dunno how I forgot that 😄